src/lookup/lookup.js

```diff
@@ -24,6 +24,10 @@ function listTimeZones () {
   return names.slice()
 }
 
+function getTimeZoneLinks () {
+  return links
```

Comment:
What about `return {  ...links }` similarly how it's `return `names.slice()` above?

Reply:
That's a good observation! I pushed a commit with the change. I didn't use the spread operator to keep in line with this project code style, that I believe doesn't use ES6 notation